src/components/CreateWiki/EditorModals/DraftModal.tsx

```diff
@@ -10,17 +10,33 @@ import {
   Button,
 } from '@chakra-ui/react'
 import { useTranslation } from 'next-i18next'
+import { useEffect, useState } from 'react'
 import { RiInformationLine } from 'react-icons/ri'
+import FeedbackModal from './FeedbackModal'
+import { useRouter } from 'next/router'
```

Comment:
```suggestion
import { RiInformationLine } from 'react-icons/ri'
```
Pls remove unused imports, also fyi linter only works on staged files in ep-ui